src/geometry/TankSceneNode.cxx

```diff
@@ -19,8 +19,14 @@
 #include "TankSceneNode.h"
 
 // system headers
+#define GLM_ENABLE_EXPERIMENTAL
```

Comment:
This should be before **any** includes. As a rule, I'm not a fan of defining macros that affect subsequent include files in the code, because you don't know what has been included that may nullify your intended effect. Rather, these types of defines should be specified on the compilation command to ensure they have the intended effect.